geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisService.java

```diff
@@ -42,6 +45,13 @@ public void close() {
     stopRedisServer();
```

Comment:
Should the `stopRedisServer` be invoked in a similar way, i.e. by listening to the cache lifecycle events?